workflow-bot-app/build.gradle

```diff
@@ -62,6 +62,7 @@ dependencies {
 
     implementation 'com.github.ben-manes.caffeine:caffeine:3.1.1'
     implementation 'io.springfox:springfox-boot-starter:3.0.0'
+    implementation 'com.google.guava:guava:31.1-jre'
```

Comment:
Does it worth adding a whole new library only for formatting the activity type name? Can't we do it with a utility function without this dependency ?

Reply:
i will have a look.

Reply:
done